site/Makefile

```diff
@@ -1,3 +1,5 @@
+# TODO: use a docker image to serve a development copy using
```

Comment:
I'm going to file a help wanted issue for this rather than do it now, this should be a good new contributor project :-)